docs/source/install/docker.rst

```diff
@@ -83,14 +85,29 @@ the containers in your installation. Best to use service discovery to set this v
 the st2client will use this configuration.
 
 ::
+
     API - TCP 9101
     Auth - TCP 9100
 
 Startup the containers with the command ``/usr/local/bin/tiller``. For example:
 
 ::
+
     docker run -d -i -t stackstorm/st2sensorcontainer /usr/local/bin/tiller
 
 Once done, see the http://docs.stackstorm.com/cli.html to configure your client to connect to API containers
 
+Note
+****
+
+With containers, supervisord_ is often used to manage processes within the container.  Since some Stackstorm components require PATH to be properly set, pay attention to appending path the right way in supervisord configuration files: use %(ENV_PATH)s instead of $PATH.
+
+::
+
+    [program:st2actionrunner]
+    command=python /opt/stackstorm/src/st2/st2actions/bin/st2actionrunner --config-file /etc/st2/st2.conf
```

Comment:
Although it does apply to the all in one container... if you use supervisord to manage all the separate processes within a single container.  Anytime supervisord is used, the comment applies.